_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,83 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is [switching back to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) on
+**DATETOBEDETERMINED**.
+
+For most users, there should be no noticeable change. If you have any packages
+installed that are no longer provided by Void, or your system has explicit
+dependencies on LibreSSL, you will of course need to take action to ensure your
+system continues to function after the switch.
+
+---
+
+The discussion started in a Request For Comments (RFC),
+[void-linux/void-packages#20935](https://github.com/void-linux/void-packages/issues/20935),
+from April 12th, 2020, and has continued with most maintainers expressing
+support for the idea. The main reasons for the switch are as listed in the RFC:
+
+- Because most software targets OpenSSL, Void will no longer have to maintain
+  (in some cases, very complex) patches to adapt packages to LibreSSL. Such
+  patches can also be risky, due to the complexity of the OpenSSL API. The
+  results can vary between runtime errors and actual security isses.
+- Extensive support for platform specific optimizations outside of x86.
+- Access to new standards and algorithms earlier, such as full TLS 1.3 support.
+
+As a result of the above, the switch to OpenSSL is expected to lessen the time
+and effort spent on packages that require it. This is especially notable
```

Comment:
Rather than "that require it" (sounds ambiguous), how about "using SSL"? (is that correct?)

Reply:
It does, so I switched to "OpenSSL-like library", not sure it's best.